Animals_class/Animal_class.py

```diff
@@ -0,0 +1,49 @@
+class Moving:
+    def move(self):
+        raise NotImplementedError(
+            'Define voice in %s.' % self.__class__.__name__)
+
+
+class Animal(Moving):
+    def voice(self):
+        super().move()
+
+
+class Transport(Moving):
+    def launch(self):
+        super().move()
+
+
+class Duck(Animal):
+    def voice(self):
+        print("duck quacks")
+
+    def move(self):
+        print("Duck is swimming")
+
+
+class Tiger(Animal):
+    def voice(self):
+        print("tiger growls")
+
+    def move(self):
+        print("Tiger is running")
+
+
+class Car(Transport):
+    def __init__(self, status="non launch"):
```

Comment:
it is not recomending define strings variables like that..
Better to use None by default and define it below using `if` operator

Reply:
But see comment below